test/integration/store.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	swarm_manage_cleanup
+	stop_docker
+	stop_store
+}
+
+# Address on which the store will listen (random port between 8000 and 9000).
+STORE_HOST=127.0.0.1:$(( ( RANDOM % 1000 )  + 8000 ))
```

Comment:
Note for later: not necessary to take a random port, it can be fixed to 8500 as we should test the scenario where the store is restarted to see if swarm behaves correctly in that case. (same as for Leader Election)